pkg_config.10.xml

```diff
@@ -928,7 +928,7 @@
 		<descr>LCD display driver</descr>
 		<website>http://www.lcdproc.org/</website>
 		<category>Utility</category>
-		<version>lcdproc-0.5.7_1 pkg.v.1.0.1</version>
+		<version>lcdproc-0.5.7_1 pkg.v.1.0.2</version>
```

Comment:
Version will also need updating in the older pkg_config etc files to keep it in sync.